CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - queue-utube handling (#85)
 - Master discovery (#113)
 - SQL support (#62)
+- Prepared statements (#117)
```

Comment:
You've also added `ExecuteAsync` and `ExecuteTyped`, this is a separate update (and it looks like it better be a separate commit.)

Reply:
Moved it to a separate commit.